### ISSUE TITLE
Extracting probabilities when using BCELossWithLogits

### ISSUE THREAD

**Issue by theravadaz** [open-mmlab/mmaction2#1856]
Hi :) I've trained a TSN and a SlowFast on a multi-label classification task. Both are trained with BCELossWithLogits. I'm trying to extract the output of the network for my test data, but am having some issues with the SlowFast.

For the TSN, the logits are all either positive or negative, and it's pretty obvious that an output value of >0 for a class is the same as the class having probability >0.5.

But for the SlowFast, the outputs are all very small positive numbers. Putting the output through a sigmoid layer just outputs 0.5 for every value. So how am I meant to extract the probabilities for each class? Is there some threshold value that the network uses at evaluation time?

Thanks very much for the help!

**Comment by theravadaz** [comment]
Sorry - to be clear, the issue seems to be that my SlowFast is not outputting logits at test time... do you have any idea what might be going on here?

**Comment by hukkai** [comment]
@theravadaz Try remove this option: https://github.com/open-mmlab/mmaction2/blob/5e09bd849d844a0f64f9866cd4f26f00e56c0086/configs/_base_/models/slowfast_r50.py#L39

**Comment by hukkai** [comment]
@theravadaz  If you have any further questions, feel free to re-open the issue. Thanks!
